testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -458,13 +469,22 @@ def _check_atomgroup(self, ag, u):
         assert_equal(len(u.atoms[4].bonds), 0)
         assert_equal(len(u.atoms[5].bonds), 0)
 
-    def test_atomgroup_guess_bonds(self):
+    @pytest.mark.parametrize(
+        'ff, lb, nbonds',
+        [
+            (0.55, 0.1, 2), (0.9, 1.6, 1),
+            (0.5, 0.2, 2), (0.1, 0.1, 0)
+        ]
+    )
+    def test_atomgroup_guess_bonds(self, ff, lb, nbonds):
         """Test an atomgroup doing guess bonds"""
         u = mda.Universe(two_water_gro)
 
         ag = u.atoms[:3]
-        ag.guess_bonds()
-        self._check_atomgroup(ag, u)
+        ag.guess_bonds(fudge_factor=ff, lower_bound=lb)
+        if ff == 0.55 and lb == 0.1:
```

Comment:
Could you add a comment why this `if` statement was added? Future developers won't necessarily understand the context.